reconcile/utils/gpg.py

```diff
@@ -34,6 +35,10 @@ def gpg_key_valid(public_gpg_key):
         if proc.returncode != 0:
             return False, out
 
+        if recepient and recepient not in str(out):
```

Comment:
And I'd very much rather this be:

```python
if recipient and recipient not in str(out):
    raise ValueError(f"{ERR_SIGNER} {recipient}")
```

Reply:
But I guess this is a useless rant right now, because we'd have to adapt other callers of this function.